src/kernels/helpers.node.ts

```diff
@@ -3,1341 +3,19 @@
 // Licensed under the MIT License.
 'use strict';
 
-import * as path from '../platform/vscode-path/path';
-import * as uriPath from '../platform/vscode-path/resources';
-import * as url from 'url';
-import type { KernelSpec } from '@jupyterlab/services';
-// eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports
-const NamedRegexp = require('named-js-regexp') as typeof import('named-js-regexp');
 import * as nbformat from '@jupyterlab/nbformat';
-// eslint-disable-next-line @typescript-eslint/no-require-imports
-import cloneDeep = require('lodash/cloneDeep');
-import {
-    IKernel,
-    KernelConnectionMetadata,
-    LocalKernelSpecConnectionMetadata,
-    LiveRemoteKernelConnectionMetadata,
-    PythonKernelConnectionMetadata,
-    IKernelProvider,
-    isLocalConnection,
-    IJupyterKernelSpec,
-    IJupyterSession,
-    KernelInterpreterDependencyResponse
-} from './types';
-import { Uri, Memento, NotebookDocument, NotebookController } from 'vscode';
-import {
-    IWorkspaceService,
-    ICommandManager,
-    IVSCodeNotebook,
-    IApplicationShell
-} from '../platform/common/application/types';
-import { PYTHON_LANGUAGE, Settings } from '../platform/common/constants';
-import { traceError, traceInfo, traceInfoIfCI, traceVerbose, traceWarning } from '../platform/logging';
+import { IKernel, KernelConnectionMetadata } from './types';
+import { Uri } from 'vscode';
+import { traceError, traceVerbose } from '../platform/logging';
 import { getDisplayPath } from '../platform/common/platform/fs-paths';
 import { IPythonExecutionFactory } from '../platform/common/process/types.node';
-import {
```

Comment:
Most of this file moved to helpers.ts. Only the 'node' specific stuff remained here.